x-pack/test/security_solution_cypress/es_archives/threat_data/mappings.json

```diff
@@ -0,0 +1,3577 @@
+{
+  "type": "index",
+  "value": {
+    "aliases": {
+      "playing": {
```

Comment:
Did you want this here where the alias is `playing`?

Reply:
I missed that, thanks @FrankHassanabad :)